plugins/node/instrumentation-fs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/instrumentation-fs",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Should not be bumped to `1.0.0`

Reply:
i'm trying to see how this happened. maybe because it is a new package, but i think 0.1.0 is supposed to be the first version